src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,53 @@
+package step2;
+
+public class Calculator {
+	private final Delimiter delimiter;
+
+	public Calculator(Delimiter delimiter) {
+		this.delimiter = delimiter;
+	}
+
+	public int calculate(String input) {
+		String[] tokens = delimiter.split(input);
+
+		int sum = 0;
+		for (String token : tokens) {
+			sum += Integer.parseInt(token);
+		}
+		return sum;
+	}
+
+	public int parseOrZero(String input) {
+		if (input.isEmpty() || input.equals("null")) {
+			return 0;
+		}
+		return Integer.parseInt(input);
+	}
+
+	public int parseIfSingleDigit(String input) {
```

Comment:
```suggestion
	String[] values = "1".split( delimiter);
```
숫자가 하나인 경우를 split할 경우 위와 같아 굳이 별도의 메서드로 분리하지 않아도 됨